app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -225,6 +227,13 @@ class PatientSummaryScreen(
         .map { PatientSummaryLinkIdCancelled }
   }
 
+  private fun phoneNumberClicks(): Observable<UiEvent> {
+    return if (config.callPatientFeatureEnabled)
+      contactTextView.clicks().map { ContactPatientClicked }
+    else
+      Observable.never()
```

Comment:
Instead of not emitting an event when click happens, we should disable click listeners. Wdyt?

Reply:
I think that makes sense, I will do this.

Reply:
Ok, I tried doing this, but it's a pain because the Rx stream will automatically make the view clickable when it sets up the stream (https://developer.android.com/reference/android/view/View.html#setOnClickListener%28android.view.View.OnClickListener%29).

Given that this is a developer flag and I am going to remove this at the end of the feature, I think I'd prefer to leave this as is for now since it's not worth the effort to fix.

Reply:
Yea. Not worth it.